### ISSUE TITLE
Apex DateTimePlugin

### ISSUE THREAD

**Issue by Macentosha** [isabolic/apex-plg-datetimepicker#3]
Hi Ivan!
I.m using Apex 5.1 and recently installed your plugin as described on github. (pkg first, plugin.sql  after pkg)
When i run a page,  and click on item there is en empty line without calendar.
I need your advise. what am i doing wrong?
thanks a lot for your help.
![image](https://user-images.githubusercontent.com/30372718/28492336-ee64efb0-6f09-11e7-8cb1-1bc046497d27.png)

**Comment by isabolic** [comment]
Hello Macentosha,

can you apply screenshot of plugin configuration from the apex page? 
Also please make screenshot of console output (F12 - developer tools), I'm guesing there is a error...

**Comment by Macentosha** [comment]
Dear Ivan, thanks for your quick response.
I did some tests and it has started work  after I set up "Format Mask" field.

![image](https://user-images.githubusercontent.com/30372718/28493046-db3a4206-6f17-11e7-864e-1caa6aaba966.png)

Then I've created another page and put  a datetimepicker from a scratch on a page. .. and It works without mask in "Format Mask" Field.

I used a console while testing. There are no errs/warns
